cfme/common/provider_views.py

```diff
@@ -16,7 +16,6 @@
                                   ItemsToolBarViewSelector,
                                   Checkbox,
                                   Input,
-                                  Table,
```

Comment:
**QUESTION**
Why is it removed?

Reply:
It's due to my flake8 git hook.
I had two option while fixing `discover_dict()` method of ec2:
1. change key of `discover_dict` from `password_verify` to `confirm_password`
2. Add changes in `CloudProvidersDiscoverView` and change `confirm_password` to `password_verify`

First I follow the second method and push changes. but found common practice is to used `confirm_password`. So revert changes but due to lint problem (l.e. table is imported but not used); I have to go with lint correction now.